config/mainCfg_ETau_UL16.cfg

```diff
@@ -0,0 +1,105 @@
+[general]
+lumi = 16800 # pb^-1 2016 postVFP
+
+data = DsingleEleF, DsingleEleG, DsingleEleH, DMETF, DMETG, DMETH
+
+signals = GGF_Radion250, GGF_Radion260, GGF_Radion270, GGF_Radion280, GGF_Radion300, GGF_Radion320, GGF_Radion350, GGF_Radion400, GGF_Radion450, GGF_Radion500, GGF_Radion550, GGF_Radion600, GGF_Radion650, GGF_Radion700,GGF_Radion750, GGF_Radion800, GGF_Radion850, GGF_Radion900, GGF_Radion1000, GGF_Radion1250, GGF_Radion1500, GGF_Radion1750, GGF_Radion2000, GGF_Radion2500, GGF_Radion3000
+
+backgrounds = TTfullyHad, TTfullyLep, TTsemiLep, DY_NLO_Incl, DY_NLO_0jet, DY_NLO_1jet, DY_NLO_2jet, DY_NLO_PtZ-0To50, DY_NLO_PtZ-50To100, DY_NLO_PtZ-100To250,DY_NLO_PtZ-250To400, DY_NLO_PtZ-400To650, DY_NLO_PtZ-650ToInf, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, WW, WZ,ZZ, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB, ttHJetToTauTau, WWW, WWZ, WZZ, ZZZ, TTWJetsToLNu, TTWJetsToQQ, TTZToLLNuNu, TTWW, TTWZ, TTZZ
+
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, bjet1_eta, bjet2_eta, tauH_mass, tauH_SVFIT_mass, tauH_pt, tauH_SVFIT_pt, bH_pt, bH_mass, bjet1_bID_deepFlavor, bjet2_bID_deepFlavor, ditau_deltaR, HH_pt, HH_mass, HHKin_mass, met_et, met_phi , DNNoutSM_kl_1, fatjet_pt, fatjet_eta, fatjet_phi, fatjet_softdropMass, fatjet_particleNetMDJetTags_score, fatjet_particleNetMDJetTags_mass, HHbregrsvfit_m, HHbregrsvfit_pt, HHbregrsvfit_eta
```

Comment:
This is the most inclusive list of variables I could put together looking at the various configs

dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, bjet1_eta, bjet2_eta, tauH_mass, tauH_SVFIT_mass, tauH_pt, tauH_SVFIT_pt, bH_pt, bH_mass, bjet1_bID_deepFlavor, bjet2_bID_deepFlavor, ditau_deltaR, HH_pt, HH_mass, HHKin_mass, met_et, met_phi , DNNoutSM_kl_1, fatjet_pt, fatjet_eta, fatjet_phi, fatjet_softdropMass, fatjet_particleNetMDJetTags_score, fatjet_particleNetMDJetTags_mass, HHbregrsvfit_m, HHbregrsvfit_pt, HHbregrsvfit_eta

dau1_phi, dau2_phi, dau1_iso, dau2_iso, dib_deltaR, btau_deltaRmin, btau_deltaRmax, HH_deltaR, dau1_deepTauVsMu, dau1_deepTauVsEle, dau1_deepTauVsJet, dau2_deepTauVsMu, dau2_deepTauVsEle, dau2_deepTauVsJet, HT20Full, metnomu_et, metnomu_phi, njets, njets20, njets50, bjet1_JER, bjet2_JER

I would suggest to remove dau1_phi, dau2_phi,btau_deltaRmin, btau_deltaRmax(I honestly do not even know what they are), HT20Full, njets20, njets50